### ISSUE TITLE
Resource links in images won't work

### ISSUE THREAD

**Issue by noisecapella** [mitodl/ocw-to-hugo#352]
Image resource links will soon be resolved to links to resource pages in #337. However this will break all images which link to these resources. For example the classroom pics on this page will be broken after #337 merges: https://ocwnext.odl.mit.edu/courses/1-00-introduction-to-computers-and-engineering-problem-solving-spring-2012/pages/instructor-insights/. To fix this I think we need to change the URL editing code to manipulate this HTML as a DOM instead of via regexes. I don't think that's a ton of work but it's enough where it should be a separate PR I think

**Comment by pdpinch** [comment]
I believe we were supposed to omit this classroom images, but I'll confirm. 

Are there other examples of links that will be broken?

**Comment by noisecapella** [comment]
@pdpinch I don't know, that requires some work to figure out across all courses

**Comment by pdpinch** [comment]
Putting this on hold until I can do some further research.

**Comment by pdpinch** [comment]
Ok, here's an example of a course image that is currently broken: https://ocwnext.odl.mit.edu/courses/es-s41-speak-italian-with-your-mouth-full-spring-2012/pages/lesson-2/

(side note: this course has a bunch of extraneous tables. I've asked the OCW team to re-author it without the tables)

**Comment by pdpinch** [comment]
Oh wait. That's a different case.
